INSTALL.md

```diff
@@ -349,6 +349,14 @@ sudo pip uninstall py-bcrypt
 sudo pip install py-bcrypt
 ```
 
+### Void Linux
+
+Synapse can be found in the void repositories as 'synapse'.  
+synchronize your repository databases and update your system to the most recent packages, including their dependencies. Then install synapse.
```

Comment:
> synchronize your repository databases and update your system to the most recent packages, including their dependencies. Then install synapse.

I assume I do this by running the commands below, so I'm not sure this sentence is adding any value. Can we get rid of it?

Reply:
Feel free to remove the sentence if you like :)